src/layers/domain/repository/cpm_product_repository/tests/v3/test_cpm_product_repository_v3.py

```diff
@@ -105,3 +106,120 @@ def test__cpm_product_repository__product_does_not_exist_local():
         )
         with pytest.raises(ItemNotFound):
             repo.read(product_team_id=product_team_id, product_id=product_id)
+
+
+@pytest.mark.integration
+def test__query_products_by_team():
```

Comment:
What's the difference between these tests and the ones in the file above?

Reply:
Ah that other file shouldn't be there. Had it before I rebased yours